static/build.js

```diff
@@ -248,24 +248,24 @@ function editNode(d) {
   d3.select('#link-menu').style('display', 'none');
   var nodeMenu = d3.select('#node-menu');
   nodeMenu.style('display', 'block');
+  document.getElementById('edit-node-name').value = d.name;
   nodeMenu.select('#edit-node-name')
-    .attr('value', d.name)
```

Comment:
Bleh, d3. I think there's a `.val()` which I missed, but yeah, this sounds like the way to go.